modules/social_features/social_profile/social_profile.module

```diff
@@ -377,6 +379,18 @@ function social_profile_preprocess_profile(array &$variables) {
       ->buildUrl($profile->field_profile_image->entity->getFileUri());
   }
 
+  // Prepare variables for statistic block.
+  $storage = \Drupal::entityTypeManager()->getStorage('node');
+  $variables['profile_topics'] = count($storage->loadByProperties([
+    'uid' => $user->id(),
+    'type' => 'topic',
+  ]));
+  $variables['profile_events'] = count($storage->loadByProperties([
+    'uid' => $user->id(),
+    'type' => 'event',
+  ]));
+  $variables['profile_groups'] = count(\Drupal::service('social_group.helper_service')->getAllGroupsForUser($user->id()));
```

Comment:
❗️  This is a performance red flag because we're loading a lot of content here to count it. We should instead just create an entity count query. Perhaps create a `UserStatistics` service much like `GroupStatistics`

Reply:
@Kingdutch 
Fixed. Created the UserStatistics service.